WDL/runtime/task.py

```diff
@@ -229,7 +231,8 @@ def _run(self, logger: logging.Logger, command: str) -> int:
         try:
             container = None
             exit_info = None
-
+            error_file = [x for x in list(volumes) if "stderr.txt" in x][0]
```

Comment:
I think this should work:

```suggestion
            error_file = os.path.join(self.host_dir, "stderr.txt")
```